scripts/infra/up_decide.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I see no reasons for this separate string cause no specific arguments needed for the canister deployment